build.xml

```diff
@@ -72,7 +72,7 @@
   <property name="jni.revision" value="1"/>
   <property name="jni.build" value="0"/> <!--${build.number}-->
   <property name="jni.version" value="${jni.major}.${jni.minor}.${jni.revision}"/>
-  <property name="jni.md5" value="1a6047467b59e8748f975e03016ce3d9"/>
+  <property name="jni.md5" value="4f72f2799dfee6008a386bc40afd7428"/>
```

Comment:
I’ll take care of the windows rebuilds.

The long name failures have been there already, they actually expose a Sun JVM bug.

> On Feb 7, 2016, at 8:56 AM, marco2357 <notifications@github.com> wrote:
> 
> In build.xml:
> 
> > @@ -72,7 +72,7 @@
> >    <property name="jni.revision" value="1"/>
> >    <property name="jni.build" value="0"/> <!--${build.number}-->
> >    <property name="jni.version" value="${jni.major}.${jni.minor}.${jni.revision}"/>
> > -  <property name="jni.md5" value="1a6047467b59e8748f975e03016ce3d9"/>
> > +  <property name="jni.md5" value="4f72f2799dfee6008a386bc40afd7428"/>
> 
> My win32 native rebuild is actually more than double the size of the previous one. Maybe you want to rebuild this yourself too.
> 
> —
> Reply to this email directly or view it on GitHub.
>